src/sql/plot.py

```diff
@@ -607,6 +618,19 @@ def _histogram_stacked(
     return data
 
 
+@modify_exceptions
+def _filter_aggregate(filter_queries):
```

Comment:
I think it's a good function and we can utilize it in the future.

A few notes on this:

1. We can move this function to [query_util.py](https://github.com/ploomber/jupysql/blob/master/src/sql/query_util.py) and rename it to something more general like `join_queries`.
2. We can change it to accept queries and not a list of queries.
3. Let's specify in the function's description that it returns a new query which is a conjunction of queries using the logical operator 'AND' (in the future we can expand it to use "OR" as well).
4. We should test this function

e.g.

```
def join_queries(*queries) -> str:
    """
    Returns a new query that is a conjunction of queries using the logical operator 'AND'
    """
    ...
```

```
    filter_query_1 = f'"{column}" IS NOT NULL'

    filter_query_2 = f"{facet['key']} == '{facet['value']}'" if facet else None

    # filter_queries = [filter_query_1, filter_query_2] <-- We don't need this step anymore
    filter_query = query_util.join_queries(filter_query_1, filter_query_2)
```

Reply:
@yafimvo 
This function works only on combining filter queries and produce an output in the form of 
`WHERE filter1 AND filter2 AND filter2 .......`

We are using it to render this templat:
```
template_ = """
    select
    floor("{{column}}"/{{bin_size}})*{{bin_size}} as bin,
    count(*) as count
    from "{{table}}"
    {{filter_query}}         <----
    group by bin
    order by bin;
    """
```
Filters here are conditions for WHERE eg:
`Columns A == NULL`
`Columns B == 2`

This function won't work for joining two independent SQL queries.

Reply:
Oh ok, got it!

so I'll update my notes:
1. Since this is a helper function for a very specific case this might be confusing in the future let's document it by adding a short description to the function of what filter query is (you can use the `Parameters` and the parameter name like [here](https://github.com/ploomber/jupysql/blob/03aa57b4d224ae72ff44ca391a9fdd528e912f89/src/sql/plot.py#L720))
2. Let's change its signature to `def _filter_aggregate(*filter_queries)` so we won't have to use a list.

Reply:
Sure! making the changes!